NGitLab.Mock/Clients/IssueClient.cs

```diff
@@ -85,12 +85,25 @@ public Models.Issue Edit(IssueEdit issueEdit)
 
                 issueToModify.Title = issueEdit.Title;
                 issueToModify.Description = issueEdit.Description;
-                issueToModify.Labels = issueEdit.Labels.Split(',');
+
+                var labelsEdit = issueEdit.Labels?.Split(',');
+
+                if (string.Equals(issueEdit.Labels, string.Empty))
+                {
+                    labelsEdit = Array.Empty<string>();
+                }
+
+                if (labelsEdit is not null)
+                {
+                    CreateResourceLabelEvents(issueToModify.Labels, labelsEdit, issueToModify.Id);
+                }
+
+                issueToModify.Labels = labelsEdit ?? issueToModify.Labels;
```

Comment:
I think you can simplify the logic to something like
```suggestion
                var labelsEdit = string.IsNullOrWhiteSpace(issueEdit.Labels) ? Array.Empty<string>() : issueEdit.Labels.Split(',');
                if (labelsEdit.Any())
                {
                    CreateResourceLabelEvents(issueToModify.Labels, labelsEdit, issueToModify.Id);
                    issueToModify.Labels = labelsEdit;
                }
```

Reply:
It's not the same behavior if it's null or if it's an empty string.

Reply:
Ok, but I'm sure you can still simplify and/or make the logic easier to understand.